models/endoscopic_inbody_classification/configs/multi_gpu_train.json

```diff
@@ -34,6 +35,8 @@
         "$@train#trainer.run()"
     ],
     "finalize": [
-        "$dist.is_initialized() and dist.destroy_process_group()"
+        "$dist.is_initialized() and dist.destroy_process_group()",
+        "$@train_close",
```

Comment:
Hi @binliunls , could you please help to review the changes of this bundle?
I just found that json files are closed and `train_close` is not used thus I add this step.

Reply:
Hi @yiheng-wang-nv ,
    Thanks for adding this. It looks good to me.

Thanks,
Bin